src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -463,26 +463,27 @@ static bool isElementNameForbidden(string name)
                         return false;
                 }
             }
+        }
 
-            // Returns 3 for "Item3".
-            // Returns -1 otherwise.
-            static int matchesCanonicalElementName(string name)
+        /// <summary>
+        /// Returns 3 for "Item3".
+        /// Returns -1 otherwise.
+        /// </summary>
+        internal static int MatchesCanonicalElementName(string name)
+        {
+            if (name.StartsWith("Item", StringComparison.Ordinal))
             {
-                if (name.StartsWith("Item", StringComparison.Ordinal))
+                string tail = name.Substring(4);
```

Comment:
Looked around but didn't find something better. Not sure I want to add one for this PR, especially since it's a pre-existing usage of `Substring` :-/

---
In reply to: [426860653](https://github.com/dotnet/roslyn/pull/44231#discussion_r426860653) [](ancestors = 426860653)